src/library/server/minauthStrategy.ts

```diff
@@ -1,3 +1,12 @@
+// TODO this module is very provisional
```

Comment:
It's been revised in my pr: https://github.com/mlabs-haskell/MinAuth/blob/846245499c8c951920b5dc27b134633d60c5f4d6/src/library/server/minauthStrategy.ts.

The new version should meet most of the requirements listed below, modulo the composability.